src/makeCalculator.js

```diff
@@ -37,7 +37,34 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    operate(callback, number) {
+      callback(number);
+
+      return this;
+    },
+    add(number) {
+      calculator.result += number;
```

Comment:
```suggestion
      this.result += number;
```

Reply:
@AlisaVlasova if I use this suggested change, I get an error " TypeError: Cannot read property 'result' of undefined".
Please refer to the screenshot:
<img width="1280" alt="error" src="https://user-images.githubusercontent.com/81584108/200270668-91db1aad-fda9-4db9-95c1-04973b050df8.png">